jsonnet/components/node-exporter.libsonnet

```diff
@@ -3,6 +3,26 @@ local textfileVolumeName = 'node-exporter-textfile';
 local tlsVolumeName = 'node-exporter-tls';
 local wtmpPath = '/var/log/wtmp';
 local wtmpVolumeName = 'node-exporter-wtmp';
+local configDir = '/var/node_exporter/config';
+local configVolumeName = 'node-exporter-config';
+local acceleratorsConfigFileName = 'accelerators-config.yaml';
+local acceleratorsConfigMapName = 'accelerators-configmap';
```

Comment:
to reduce the risk of any collision and to provide as much context as possible I'd go with `node-exporter-accelerators-collector-config` (this is also consistent with other naming)

Reply:
Fixed